lexicons/com/atproto/identity/resolveHandle.json

```diff
@@ -7,6 +7,7 @@
       "description": "Provides the DID of a repo.",
       "parameters": {
         "type": "params",
+        "required": ["handle"],
         "properties": {
           "handle": {
```

Comment:
should probs drop the "if not specified" bit of the description (at least it was still in the comment)